src/js/helpers.js

```diff
@@ -457,27 +457,34 @@ export const standardizePhases = phases =>
   });
 
 export const getDebConfigurationCode = (ipAddress, isHosted, isEnterprise, tenantToken, deviceType = 'generic-armv6', isPreRelease) => {
+  let envVars = ``
   let installScriptArgs = `--demo`;
   if (isPreRelease) {
     installScriptArgs = `${installScriptArgs} -c experimental`;
   }
   if (isHosted) {
     const jwtToken = getToken();
-    installScriptArgs = `${installScriptArgs} --commercial --jwt-token "${jwtToken}"`;
+    envVars = `${envVars}JWT_TOKEN="${jwtToken}"
+`
```

Comment:
```suggestion
    envVars = `${envVars}JWT_TOKEN="${jwtToken}"\n`;
```
for when you're fixing the linter errors 😉

Reply:
:+1: